### DISCUSSION TITLE
More recent tag of "jree"

### DISCUSSION THREAD

**Discussion by aasselin-en** [mike-lischke/jree#5] [General]
Hi,

we recently discovered your tool to convert Java source code to Typescript, we fall on a few problems that could be resolved by gaining access to the latest source code of "jree" (we are going to use the tgz in the meanwhile)
Would you publish a more recent tagged version of "jree" (for example, we needed String.startsWith)?

Thank you for your great work!

Regards
Armel

**Comment by mike-lischke** [comment]
@aasselin-en I moved this discussion to the right repo (jree) and created an issue (#6) out of it.

**Reply by aasselin-en** [reply]
thanks
